legend-sdlc-server-fs/src/main/java/org/finos/legend/sdlc/server/startup/AbstractBaseModule.java

```diff
@@ -0,0 +1,474 @@
+import org.finos.legend.sdlc.server.resources.revision.project.group.GroupWorkspaceRevisionsResource;
+import org.finos.legend.sdlc.server.resources.revision.project.user.WorkspaceEntityRevisionsResource;
+import org.finos.legend.sdlc.server.resources.revision.project.user.WorkspacePackageRevisionsResource;
+import org.finos.legend.sdlc.server.resources.revision.project.user.WorkspaceRevisionsResource;
+import org.finos.legend.sdlc.server.resources.user.CurrentUserResource;
+import org.finos.legend.sdlc.server.resources.user.UsersResource;
+import org.finos.legend.sdlc.server.resources.version.VersionsResource;
+import org.finos.legend.sdlc.server.resources.workflow.ReviewWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.ReviewWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.VersionWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.VersionWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.PatchReviewWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.PatchReviewWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.PatchWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.PatchWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.group.PatchesGroupWorkspaceWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.group.PatchesGroupWorkspaceWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.user.PatchesWorkspaceWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.patch.user.PatchesWorkspaceWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.ProjectWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.ProjectWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.group.GroupWorkspaceWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.group.GroupWorkspaceWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.user.WorkspaceWorkflowJobsResource;
+import org.finos.legend.sdlc.server.resources.workflow.project.user.WorkspaceWorkflowsResource;
+import org.finos.legend.sdlc.server.resources.workspace.patch.group.PatchesGroupWorkspacesResource;
+import org.finos.legend.sdlc.server.resources.workspace.patch.user.PatchesWorkspacesResource;
+import org.finos.legend.sdlc.server.resources.workspace.project.group.GroupWorkspacesResource;
+import org.finos.legend.sdlc.server.resources.workspace.project.user.WorkspacesResource;
+import org.finos.legend.sdlc.server.tools.BackgroundTaskProcessor;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public abstract class AbstractBaseModule extends DropwizardAwareModule<LegendSDLCServerFSConfiguration>
```

Comment:
I don't see why this class exists separately from BaseModule. Is it being extended by any other class?